### PR TITLE
team_info passes spec_harness

### PR DESCRIPTION
**Pull request by whitneykidd**
Updated team_info and tests to pass spec_harness

### CODE REVIEW

File: lib/team_stats.rb [hunk 1]
```diff
@@ -6,17 +6,17 @@ def initialize(games, teams, game_teams)
   end
 
   def team_info(team_id)
+    team = @teams.select { |team| team.team_id.to_s == team_id }.first
```
Comment (Maxwell-Baird) [inline lib/team_stats.rb:9]: Smart thinking here in order to get the right team

File: lib/team_stats.rb [hunk 2]
```diff
@@ -6,17 +6,17 @@ def initialize(games, teams, game_teams)
...
     {
-      "team_id" => team_id,
-      "franchiseid" => @teams[team_id].franchise_id.to_i,
-      "teamname" => @teams[team_id].teamname,
-      "abbreviation" => @teams[team_id].abbreviations,
-      "link" => @teams[team_id].link
+      "team_id" => team_id.to_s,
+      "franchise_id" => team.franchise_id.to_s,
```
Comment (tylertomlinson) [inline lib/team_stats.rb:12]: Weird how data is a string in the csv

File: lib/team_stats.rb (hunk 1 above)
Comment (alex-latham) [inline lib/team_stats.rb:9]: Candidate for moving into a common helper method

**Comment by tylertomlinson** [comment]
Nice changes to get it passing.

**Pull request merged**